src/wordlookup/wiktionaryel.py

```diff
@@ -43,11 +47,16 @@ class WiktionaryQuery:
     __slots__ = "word", "soup"
 
     @classmethod
-    async def create(cls, word: str):
+    async def create(cls, word: str, language: str):
```

Comment:
You probably want language as an attribute of the instance instead of passing it to every method later on.